lib/ui/painting.dart

```diff
@@ -3974,22 +3974,33 @@ class Vertices extends NativeFieldWrapperClass1 {
 
   /// Creates a set of vertex data for use with [Canvas.drawVertices], directly
   /// using the encoding methods of [new Vertices].
+  /// Note that this constructor uses a raw typed data list,
+  /// so it runs faster than Dart list version 
```

Comment:
faster than the [Vertices()] constructor

(Make sure it links correctly to the regular constructor)